### PR TITLE
If less than 3 storage nodes, increase number of disks

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: https://github.com/rjfd/sesdev/issues/20

### CODE REVIEW

**Comment by rjfd** [comment]
Instead of calculating the number of storage nodes while parsing the roles string, how about having a function that calculates the number of storage nodes given the roles list (the result of the roles parsing function)?

That way we can avoid that global variable...

File: sesdev/__init__.py [hunk 1]
```diff
@@ -215,6 +220,14 @@ def create():
     """
 
 
+def _count_storage_nodes(roles):
+    storage_nodes = 0
+    for node in roles:
+        if 'storage' in node:
+            storage_nodes += 1
+    return storage_nodes
```
Comment (rjfd) [inline sesdev/__init__.py]: What about?

```python
return len([node for node in roles if 'storage' in n])
```

Comment (smithfarm) [reply sesdev/__init__.py]: That's beyond my comprehension level. What does `n` mean in this context?

Comment (rjfd) [reply sesdev/__init__.py]: it's a bug :) edited to fix it

Comment (rjfd) [reply sesdev/__init__.py]: it creates a list of nodes that have the 'storage' role and then returns the length of the list

**Comment by smithfarm** [comment]
@rjfd Ready to go now (I found it had been breaking the case when the user does not provide either `--roles` or `--single-node` - fixed)

**Comment by smithfarm** [comment]
Oops, this PR introduced a regression, #25, which is fixed by #26

**Pull request merged**